docs/src/modules/ROOT/pages/constraints-and-score/score-calculation.adoc

```diff
@@ -1648,48 +1525,8 @@ public interface ConstraintMatchAwareIncrementalScoreCalculator<Solution_, Score
 }
 ----
 
-For example in machine reassignment, create one `ConstraintMatchTotal` per constraint type and call `addConstraintMatch()` for each constraint match:
```

Comment:
Can we keep or replace the code by another example?

Reply:
I discussed this with Geoffrey and we agreed that this would not be replaced.